addons/binding/org.openhab.binding.kodi/src/main/java/org/openhab/binding/kodi/internal/model/KodiPVRChannel.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.kodi.internal.model;
+
+/**
+ * Class representing a Kodi PVR channel
+ *
+ * @author Christoph Weitkamp - Initial contribution
+ */
+public class KodiPVRChannel extends KodiBaseItem {
+    /**
+     * The PVR channel id
+     */
+    private int channelId;
```

Comment:
Very minor, but noting slight inconsistency in naming. Here it is `channelId` and `getId`. Elsewhere it's `channelID` and `getChannelID`. I might not even bring this up, except that elsewhere in the code `channelId` refers to the ESH `channelId`.

Reply:
I thought about it and feel that my naming is OK in thus place. But I changed it in the rest of the binding to have a clear line when we talk about an ESH channel or a TV channel.